common/src/main/java/org/apache/comet/parquet/NativeBatchReader.java

```diff
@@ -459,7 +459,9 @@ public void close() throws IOException {
       importer = null;
     }
     nativeUtil.close();
-    Native.closeRecordBatchReader(this.handle);
+    if (handle > 0) {
+      Native.closeRecordBatchReader(this.handle);
```

Comment:
nit: we refer to both `handle` and `this.handle`. I assume they are the same thing but it would be nice to make them consistent

Reply:
Thanks, changed